### ISSUE TITLE
ShortCut Icon Tooltip 

### ISSUE THREAD

**Issue by Naganathan** [oleg-shilo/wixsharp#1459]
Is there any option to update ShortCut icon tooltip text when creating a MSI using wixsharpextension like below image?

![image](https://github.com/oleg-shilo/wixsharp/assets/5599700/d9b7707a-af8a-4714-8043-095b36a934bd)

**Comment by oleg-shilo** [comment]
It is something that is controlled by OS. OS simply shows the name of the shortcut file without the extension.

**Comment by Torchok19081986** [comment]
morning , i found in wix V4 following : https://wixtoolset.org/docs/schema/wxs/shortcut/
Maybe it helps.

**Comment by oleg-shilo** [comment]
Yeah, I reviewed the schema already. It has nothing related to the file tooltip. 
It's all controlled by the OS not by the msi.

**Comment by oleg-shilo** [comment]
The shortcut tootip is the "Comment" property of the shortcut file:
![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/f26b36ca-45e4-44e8-aeab-ee73368881b7)

The WiX spec for the Shortcut element has no such attribute. Tough quite possibly the attribute `Description` will be mapped to the `Comment` property. WiX documentation is silent about the purpose of the `Description` attribute.
You just need to test it and see.

```C#
new FileShortcut("MyApp", "INSTALLDIR")
{
    AttributesDefinition = "Description=My Tooltip"
},
```

**Comment by oleg-shilo** [comment]
Well, indeed it is the solution you are looking for :)

![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/fb47206b-a938-44a6-9145-6b3f0da3ebcd)

**Comment by oleg-shilo** [comment]
I have added now a new property `Shortcut.Description`. But until it is available in the next release please use this technique to set the attribute:

```C#
new FileShortcut("MyApp", "%Desktop%")
{
    AttributesDefinition = "Description=My Tooltip"
}
```

**Comment by Naganathan** [comment]
Hi @oleg-shilo 

Thanks for your update. The suggested code changes working fine in my project. 
`
new FileShortcut("MyApp", "%Desktop%")
{
    AttributesDefinition = "Description=My Tooltip"
}
`

**Comment by Naganathan** [comment]
Also, please let me know what is property to provides product name to display in control panel's Add/Remove programs by using Bundle() code implementation.

![image](https://github.com/oleg-shilo/wixsharp/assets/5599700/691b6158-dd0b-479f-9dec-2461aaaa40a6)

**Comment by oleg-shilo** [comment]
Control panel info is controlled by the `Project.ControlPanelInfo` properties:
![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/0b2ad31d-836a-4e71-b14a-1b5343823f49)

But the name filed is most likely coming from the SMI product name.
